scripts/autolabels.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+ : <<'BASH_MODULE_DOCS'
+
+This file contains the dictionary for converting a modified dir into a github label,
+as well as the script to perform the conversion.
+
+Rules:
+* only lines starting with `|t-` are considered;
+* repeated spaces, commas and pipes (|) are ignored;
+* `Mathlib` is added by default to each `Root folder`.
+
+Exception: `Tactic/Linter` is converted into `Tactic-Linter` internally, to simplify separating it from
+`Tactic/anything_other_than_linter`.
+
+|Label                   |Root folders                                                                       |
+|-                       |-                                                                                  |
+|t-algebra               | Algebra, FieldTheory, RingTheory, GroupTheory, RepresentationTheory, LinearAlgebra|
+|t-algebraic-geometry    | AlgebraicGeometry, Geometry/RingedSpace                                           |
+|t-euclidean-geometry    | Geometry.Euclidean                                                                |
```

Comment:
Is this a typo, and should be `Geometry/Euclidean`?

Reply:
Indeed, it is a typo!  However, it would have probably matched regardless, since the matching is via regex and `.` matches anything!  :smile: